### PR TITLE
Fix runtime crash

### PR DESCRIPTION
**Pull request by jkim-julie**
For now, USE_OZONE will use InputMethodMinimal.

### CODE REVIEW

File: src/chrome/browser/chrome_content_browser_client.cc [hunk 1]
```diff
@@ -1081,9 +1081,7 @@ content::BrowserMainParts* ChromeContentBrowserClient::CreateBrowserMainParts(
   // Construct additional browser parts. Stages are called in the order in
   // which they are added.
 #if defined(TOOLKIT_VIEWS)
-  // Removed !defined(USE_OZONE) for ozone wayland external port
-  // ozone port for wayland should use ChromeBrowserMainExtraPartsViewsLinux
-#if defined(OS_LINUX) && !defined(OS_CHROMEOS)
+#if defined(OS_LINUX) && !defined(OS_CHROMEOS)  && !defined(USE_OZONE)
```
Comment (msisov) [inline src/chrome/browser/chrome_content_browser_client.cc]: #if defined(OZONE_PLATFORM_WAYLAND_EXTERNAL)  || (defined(OS_LINUX) && !defined(OS_CHROMEOS) && !defined(USE_OZONE)) ? And I believe this must go into a separate patch.

Comment (msisov) [reply src/chrome/browser/chrome_content_browser_client.cc]: and let's separate these two commits as well

Comment (jkim-julie) [reply src/chrome/browser/chrome_content_browser_client.cc]: Updated patch with the separate definition with OZONE_PLATFORM_WAYLAND_EXTERNAL and USE_OZONE and made two patches.
